src/Giraffe.ViewEngine/Engine.fs

```diff
@@ -634,14 +634,24 @@ module RenderView =
 
     [<RequireQualifiedAccess>]
     module AsBytes =
+
+        #if !NETSTANDARD2_0
         open System.Buffers
+        #endif
 
         let private outputAsBytes (sb : StringBuilder) =
-            let chars = ArrayPool<char>.Shared.Rent sb.Length
+            let chars = 
+                #if NETSTANDARD2_0
+                Array.create (sb.Length) Char.MinValue
+                #else
+                ArrayPool<char>.Shared.Rent sb.Length
```

Comment:
Only Suggestion I have is if we wanted to remove the ifdef around the open statement, we could fully qualify here and in the next call.  That's my only suggestion, feel free to ignore it.

Reply:
Done ;)